Backend/Backend.UnitTest/EnterpriseControllerTest.cs

```diff
@@ -62,8 +62,14 @@ public void GetEnterprises_ReturnsEnterprisesInAlphabeticalOrderByName()
         [TestMethod]
         public void GetEnterprise_ReturnsSingleEnterprise()
```

Comment:
See the changes below - once you make them you may want to consider writing some more tests, with things like making sure the correct exception is thrown if the enterprise ID doesn't exist or if the password is wrong, and making sure that all the enterprise details (including participants, places, etc) get mapped correctly in the object graph that gets returned.